src/library_webgl2.js

```diff
@@ -1116,28 +1116,28 @@ var LibraryWebGL2 = {
   glClearBufferfi__sig: 'viifi',
 };
 
-// Simple pass-through functions. Starred ones have return values. [X] ones have X in the C name but not in the JS name
-var webgl2Funcs = [[0, 'endTransformFeedback pauseTransformFeedback resumeTransformFeedback'],
- [1, 'beginTransformFeedback readBuffer endQuery'],
- [2, ''],
- [3, ''],
- [4, 'clearBufferfi'],
- [5, 'vertexAttribI4i vertexAttribI4ui copyBufferSubData texStorage2D renderbufferStorageMultisample'],
- [6, 'texStorage3D'],
- [7, ''],
- [8, ''],
- [9, 'copyTexSubImage3D'],
- [10, 'blitFramebuffer']];
-
 #if MAX_WEBGL_VERSION >= 2
```

Comment:
This entire library is only included from `modules.js` if that condition is true, so I don't think we need it here.

Reply:
I think maybe you are right, but I'm not 100% sure.   Maybe a followup?

Reply:
Sounds good.